packages/p2p/src/components/app.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter, Switch, Route, Link } from 'react-router-dom';
+import Ads from './ads/ads.jsx';
+import Orders from './orders/orders.jsx';
+import './app.scss';
+
+const App = () => (
+    <BrowserRouter>
```

Comment:
Will the P2P package completely handle its own routing? Let's have a discussion for this.